### PR TITLE
Mobile UI: Navbar with Branding has layout issues

### PR DESCRIPTION
**Pull request by josh-willis-arcadis**
add conditional render for branding element on mobile widths.

This is a stop gap for now. There is a TODO comment in app-frame.tsx to add the mobile view header.

**Description:**

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/app/desktop-nav.tsx [hunk 1]
```diff
@@ -104,16 +105,21 @@ const DesktopNav = ({
...
-            <BrandingElement
-              className={branding && `with-icon icon-${branding}`}
-              {...brandingProps}
-            >
-              {/* A title is always rendered (e.g.for screen readers)
-                  but is visually-hidden if a branding icon is used. */}
-              <div className="navbar-title">{title}</div>
-            </BrandingElement>
+
+            {!isMobile() && (
```
Comment (binh-dam-ibigroup) [inline lib/components/app/desktop-nav.tsx:109]: This removes the logo while in the account page. Is that the desired effect?

File: lib/components/app/desktop-nav.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { accountLinks, getAuth0Config } from '../../util/auth'
 import { AppConfig } from '../../util/config-types'
 import { AppReduxState } from '../../util/state-types'
 import { DEFAULT_APP_TITLE } from '../../util/constants'
+import { isMobile } from '@opentripplanner/core-utils/lib/ui'
```
Comment (binh-dam-ibigroup) [inline lib/components/app/desktop-nav.tsx:12]: Install the ESLint extension for your IDE and address the warning about import order.

**Review by binh-dam-ibigroup — COMMENTED** [review]
Are the client ok with removing the logo while in the account page?

File: lib/components/app/desktop-nav.tsx (hunk 1 above)
Comment (josh-willis-arcadis) [reply lib/components/app/desktop-nav.tsx:109]: Yes. The logo is not shown in the nav bar anywhere on the site while on mobile. To maintain consistency we will need to add the H1 tag in the navbar later down the road. The TODO comment in app-frame.tsx is where that logic will need to go.

**Comment by josh-willis-arcadis** [comment]
> Are the client ok with removing the logo while in the account page?
The only things mentioned by the client in the ticket was the wrapping issue, and the inconsistency in the logo being displayed in some places and not others.

**Review by binh-dam-ibigroup — APPROVED** [review]
One tiny thing then it should be good to go.

**Review by daniel-heppner-ibigroup — APPROVED** [review]
LGTM! Congrats on first PR!

**Pull request merged**